newton/_src/utils/import_usd.py

```
@@ -1707,21 +1948,71 @@ def parse_merged_joints(
 
             limit_key = "limit_angular" if is_revolute else "limit_linear"
-            limit_ke, limit_ke_source = _resolve_joint_limit_gain(
+            j_newton_limit_ke, j_limit_ke_resolver = R.get_value_with_resolver(
+                jp_prim, prim_type=PrimType.JOINT, key="limit_ke", default=None, verbose=verbose
+            )
+            j_newton_limit_kd, j_limit_kd_resolver = R.get_value_with_resolver(
+                jp_prim, prim_type=PrimType.JOINT, key="limit_kd", default=None, verbose=verbose
+            )
+            if j_limit_ke_resolver is not None and j_limit_ke_resolver.name == "mjc":
+                j_limit_ke_is_mjc = "authored"
+                j_limit_ke_priority = _mjc_priority
```

Ruff (0.15.20)

[error] Undefined name `_mjc_priority`

(F821)